src/components/App/App.tsx

```diff
@@ -9,23 +10,43 @@ import Journey from '../Journey/Journey';
 import {Survey} from '../Survey/Survey';
 import {Question} from '../Question/Question';
 import GenerateReport from '../GenerateReport/GenerateReport';
+import Error from '../Error/Error';
 //create interface for context
 
 const App:React.FC = () =>{
-  const [questions, updateQuestions] = useState<AllQuestionFormat | null>(null);
+  const [questions, updateQuestions] = useState<AllQuestionFormat | {}>({});
+  const [answers, updateAllAnswers] = useState<any>({});
+
```

Comment:
Personally, I  like the way the reduce method is used to create our answers object.